tests/test_config_consistency.py

```diff
@@ -71,3 +72,86 @@ def test_status_code_202(self):
         assert spans[0]["type"] == "web"
         assert spans[0]["meta"]["http.status_code"] == "202"
         assert spans[0]["error"] == 1
+
+
+@scenarios.default
+@features.tracing_configuration_consistency
+class Test_Config_HttpClientErrorStatuses_Default:
+    """ Verify behavior of http clients """
+
+    def setup_status_code_400(self):
+        self.r = weblog.get("/make_distant_call", params={"url": "http://weblog:7777/status?code=400"})
+
+    def test_status_code_400(self):
+        assert self.r.status_code == 200
+        content = json.loads(self.r.text)
+        assert content["status_code"] == 400
+
+        interfaces.library.assert_trace_exists(self.r)
+        spans = [s for _, _, s in interfaces.library.get_spans(request=self.r, full_trace=True)]
+
+        client_span = _get_span_by_name(spans, "http.request")
```

Comment:
In python the name of the span will be `requests.request`. Can we instead get the span by resource name and span kind (ie match `GET /` and `client`). I think this value should be consistent across languages.

DD_TRACE_SPAN_ATTRIBUTE_SCHEMA is v0 by default in python.

Reply:
Ex: https://github.com/DataDog/system-tests/pull/3026/files#diff-4eba22e107ccb54fd5d8f0acdaf5c8eb131320d4cfbae71c4d664a4616eb1b15R102

Reply:
Sounds good, will provide an update soon

Reply:
Implemented in 6fb43882ec7a7490c3d1d75fac41258f51d33dab